pkg/statsd/receiver.go

```diff
@@ -54,21 +59,28 @@ func (mr *MetricReceiver) RunMetrics(ctx context.Context) {
 		case <-ctx.Done():
 			return
 		case <-ticker.C:
-			mr.statser.Count("packets_received", float64(atomic.SwapUint64(&mr.packetsReceived, 0)), nil)
+			packetsReceived := float64(atomic.SwapUint64(&mr.packetsReceived, 0))
+			mr.statser.Count("packets_received", packetsReceived, nil)
 			mr.statser.Count("metrics_received", float64(atomic.SwapUint64(&mr.metricsReceived, 0)), nil)
 			mr.statser.Count("events_received", float64(atomic.SwapUint64(&mr.eventsReceived, 0)), nil)
 			mr.statser.Count("bad_lines_seen", float64(atomic.SwapUint64(&mr.badLines, 0)), nil)
+			mr.statser.Gauge("avg_packets_in_batch", packetsReceived/float64(atomic.SwapUint64(&mr.batchesRead, 0)), nil)
```

Comment:
@tiedotguy I went with avg not max... because it was a lot easier to calculate with atomic operations and I think it'll give what we're after

Reply:
```INFO[0009] stats.gauge.statsd.avg_packets_in_batch.s.covfefe NaN 1506714688```

Needs a check for zero.